docs/reference/quickstart/full-text-filtering-tutorial.asciidoc

```diff
@@ -0,0 +1,588 @@
+[[full-text-filter-tutorial]]
+== Basic full-text search and filtering in {es}
+++++
+<titleabbrev>Basics: Full-text search and filtering</titleabbrev>
+++++
+
+This is a hands-on introduction to the basics of full-text search with {es}, also known as _lexical search_, using the <<search-search,`_search` API>> and <<query-dsl,Query DSL>>.
```

Comment:
Philosophical question here, but should we refactor new tutorials to use retrievers now that they will be GA soon?

Reply:
Maybe future tutorials, honestly I'd like an aligned answer from product because the query languages decision for new users is getting complex between query DSL / retrievers / ES|QL